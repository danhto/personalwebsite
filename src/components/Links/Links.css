.links-content {
    padding-top: 3%;
}

.links-header {
    padding-bottom: 3%;
    border-bottom: 1px black solid;
}

.links-linkedin {
    font-size: 30px;
}

.links-github {
    font-size: 30px;
}