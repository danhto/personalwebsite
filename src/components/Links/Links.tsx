import React from 'react'
import {MDBBtn, MDBContainer} from 'mdbreact'

const Links = () => {

    return (
        <MDBContainer className="links-content">
            <h2 className="links-header">Want to know even more? <br/>
                Try the links below if you couldn't find your answer here! <br/>
                Thanks for visiting!</h2>
            <MDBBtn className="links-linkedin" color="primary" href="https://www.linkedin.com/in/danhto">
                LinkedIn
            </MDBBtn>
            <br/>
            <MDBBtn className="links-github" color="secondary" href="https://github.com/danhto">
                Github
            </MDBBtn>
        </MDBContainer>
    )
    
}

export default Links