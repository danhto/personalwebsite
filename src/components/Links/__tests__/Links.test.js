import {configure, shallow} from 'enzyme';
import React from 'react';
import Links from '../Links';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

describe('<Links /> component tests', () => {
    it('expect component to mount', () => {
        const about = shallow(<Links />);
        expect(about).toBeTruthy();
    });
    
    it('match snapshot', () => {
        const about = shallow(<Links />);
        expect(about).toMatchSnapshot();
    })
});