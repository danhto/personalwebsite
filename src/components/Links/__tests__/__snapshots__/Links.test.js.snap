// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Links /> component tests match snapshot 1`] = `ShallowWrapper {}`;
